src/sass/_common/wallet_header.scss

```diff
@@ -0,0 +1,226 @@
+            padding: 12px;
+            border-bottom: 1px solid var(--general-hover);
+
+            &--text {
+                font-size: $FONT_SIZE_S;
+                font-weight: bold;
+                line-height: 1.5;
+            }
+            img {
+                width: 16px;
+                height: 16px;
+            }
+        }
+        &-footer {
+            display: flex;
+            font-size: $FONT_SIZE_S;
+            justify-content: space-between;
+            padding: 16px;
+            line-height: 27px;
+
+            & > span {
+                cursor: pointer;
+            }
+        }
+        &-icon {
+            &--container {
+                align-items: center;
+                border-radius: 4px;
+                justify-content: center;
+                height: 40px;
+                width: 64px;
+            }
+        }
+    }
+}
+
+.mobile {
+    &__wallet-platform-switcher {
+        &-current {
+            height: 72px;
+            display: flex;
+            justify-content: center;
+            border-bottom: 1px solid var(--general-section-2);
+            align-items: center;
+            padding: 0 32px;
+            cursor: pointer;
+        }
+        &-container {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            gap: 8px;
+        }
+        &-logo {
+            width: 148px !important;
+            height: 32px !important;
+        }
+    }
+}
+
+@media only screen and (max-width: 879px) {
+    .wallet {
+        &__header {
+            .header__notification {
+                order: 2;
+            }
+        }
+        &__switcher {
+            background: var(--general-main-2);
+            box-shadow: 0 8px 16px 0 var(--shadow-menu);
+
+            &-accounts {
+                &-list {
+                    border-bottom: none;
+                    max-height: 100%;
+                }
+                &-btn {
+                    width: 100%;
+                    height: 40px;
+
+                    &-container {
+                        margin: 10px 16px;
+                        display: flex;
+                    }
+                }
+                &-container {
+                    display: flex;
+                    flex-direction: column;
+                    justify-content: space-between;
+                    height: calc(100% - 107px);
+                }
+            }
+            &-dropdown {
+                width: 100vw;
+                height: 100vh;
+                top: 0;
+                left: 0;
+                transform: unset;
+                z-index: 99999;
+                display: none;
+
+                &--show {
+                    display: block;
+                }
+            }
+            &-footer {
+                border-top: 1px solid var(--general-hover);
+                bottom: 0;
+                position: fixed;
+                width: stretch;
```

Comment:
First time saw this css property for width))))

Reply:
i didnt know until yesterday either